apps/user_ldap/ajax/testConfiguration.php

```diff
@@ -22,14 +23,18 @@
 
 
 try {
-	$configurationOk = true;
+	$configurationError = '';
 	$conf = $connection->getConfiguration();
 	if ($conf['ldap_configuration_active'] === '0') {
 		//needs to be true, otherwise it will also fail with an irritating message
 		$conf['ldap_configuration_active'] = '1';
-		$configurationOk = $connection->setConfiguration($conf);
 	}
-	if ($configurationOk) {
+	try {
+		$connection->setConfiguration($conf, throw:true);
```

Comment:
```suggestion
		$connection->setConfiguration($conf, throw: true);
```

I expected linter to ask for ^ but it does not :astonished:

Reply:
PR on coding-standard welcome :-P